apps/grant/src/index.js

```diff
@@ -24,16 +24,12 @@ const vendorAppJson = {
                   "required": true,
                   "validation": "[a-fA-F0-9]",
                },
-            },
-         },
-         "column1": {
-            "children": {
                "textFormField": {
                   "label": "Phone Number",
                   "validation": "[0-9]",
                },
                "button": {
-                  "action": "submit",
+                  "action": "Click Me!",
```

Comment:
Is this just the text in the button? I thought that was separate from the action

Reply:
I think submit is also still better than click me